src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -541,9 +541,6 @@ protected function parse()
             $showDelete = false;
         }
 
-        // show delete button
-        $this->tpl->assign('showPagesDelete', $showDelete);
```

Comment:
this isn't the fix, you should use a different template variable name for this. you are changing the intended implementation